ena-submission/src/ena_deposition/ena_submission_helper.py

```diff
@@ -744,11 +751,12 @@ def set_error_if_accession_not_exists(
     accession: str,
     accession_type: Literal["BIOPROJECT"] | Literal["BIOSAMPLE"],
     db_pool: SimpleConnectionPool,
+    config: Config,
 ) -> bool:
     """Make request to ENA to check if an accession exists"""
     url = f"https://www.ebi.ac.uk/ena/browser/api/summary/{accession}"
-    response = requests.get(url, timeout=10)
     try:
+        response = ena_http_get_with_retry(config, url)
```

Comment:
I think this function can always be retried if it times out as it just checks if the erz accession has been accessioned